doc/snippets/Microsoft.Data.SqlClient/SqlCommand.xml

```diff
@@ -14,7 +14,7 @@ When an instance of <xref:Microsoft.Data.SqlClient.SqlCommand> is created, the r
 
 |Item|Description|
 |----------|-----------------|
-|<xref:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteNonQuery%2A>|Initiates the asynchronous execution of the Transact-SQL statement or stored procedure that is described by this <xref:Microsoft.Data.SqlClient.SqlCommand>, generally executing commands such as INSERT, DELETE, UPDATE, and SET statements. Each call to <xref:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteNonQuery%2A> must be paired with a call to <xref:Microsoft.Data.SqlClient.SqlCommand.EndExecuteNonQuery%2A> which finishes the operation, typically on a separate thread.|
+|<xref:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteNonQuery%2A>|Initiates the asynchronous execution of the Transact-SQL statement or stored procedure that is described by this <xref:Microsoft.Data.SqlClient.SqlCommand>, generally executing commands such as INSERT, DELETE, UPDATE, and SET statements. Each call to <xref:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteNonQuery%2A> must be paired with a call to <xref:Microsoft.Data.SqlClient.SqlCommand.EndExecuteNonQuery> which finishes the operation, typically on a separate thread.|
```

Comment:
```suggestion
|<xref:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteNonQuery>|Initiates the asynchronous execution of the Transact-SQL statement or stored procedure that is described by this <xref:Microsoft.Data.SqlClient.SqlCommand>, generally executing commands such as INSERT, DELETE, UPDATE, and SET statements. Each call to <xref:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteNonQuery> must be paired with a call to <xref:Microsoft.Data.SqlClient.SqlCommand.EndExecuteNonQuery> which finishes the operation, typically on a separate thread.|
```

Reply:
Suggested change removed `%2A` from `xref:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteNonQuery`

Reply:
Actually is there a reason why the `%2A` is removed? I noticed some are removed and some are not.

Reply:
Its kind of a mix actually. I'm figuring out what actually is needed. As putting all these %2A lands up into problems sometimes.

Reply:
ya it's strange why `BeginExecuteNonQuery` is ok with %2A, but `EndExecuteNonQuery` is not... but if it doesn't cause any error, let's just leave it. You can close my other similar comments too.